okhttp/src/jvmMain/kotlin/okhttp3/internal/http2/flowcontrol/WindowCounter.kt

```diff
@@ -0,0 +1,37 @@
+package okhttp3.internal.http2.flowcontrol
+
+import java.util.concurrent.atomic.AtomicLong
+
+class WindowCounter(
+  val streamId: Int
+) {
+  /** The total number of bytes consumed. */
+  private val _total: AtomicLong = AtomicLong(0)
```

Comment:
Why atomics?

Reply:
I don't think I'm clear on all the locking that goes on in all cases. I originally had a data class that was updated. Might switch back to that and define locking rules around writes.

Reply:
Removed, no need.